app/lib/ui/views/setting_screen.dart

```diff
@@ -67,7 +67,20 @@ class _SettingScreenState extends State<SettingScreen> {
                         ),
                         RaisedButton.icon(
                           //Just added model.signOut for now, will remove once created model for about screen
-                          onPressed: model.signOut,
+                          onPressed: () => showLicensePage(
+                            context: context,
+                            applicationName: "Humane Transport",
+                            applicationIcon: Padding(
+                                padding: EdgeInsets.all(9),
+                                child: Image.asset('assets/splash_screen.png',
+                                    width: 60, height: 60)),
+                            applicationVersion: 'Version 0.0.1',
+                            applicationLegalese:
+                                'Copyright (C) 2020 Clark Inocalla, Kelly Holtzman, Mansi Patel, Sana Khan. '
```

Comment:
Looks very good! I would put more space in the paragraph to look a bit nicer.

- [ ] Make the `applicationLegalese` look nicer to read

Reply:
@holtzmak I have looked into this. Because this is their in-built page, so we kind of not able to modify this. We ca just pass the string into the function.

Reply:
Right, we can break the string using inline `\n` though! Could you give that a try?

Reply:
@holtzmak Is this okay? This is all I can do because there is no custom page option available for this page at this point. 
![image](https://user-images.githubusercontent.com/44125953/113471524-2a058f00-941a-11eb-84dc-e1c06a8232a6.png)

Reply:
Yeah that's great!